packages/beacon-node/test/unit/util/sszBytes.test.ts

```diff
@@ -15,26 +16,38 @@ import {
 } from "../../../src/util/sszBytes.js";
 
 describe("attestation SSZ serialized picking", () => {
-  const testCases: phase0.Attestation[] = [
+  const testCases: allForks.Attestation[] = [
     ssz.phase0.Attestation.defaultValue(),
     attestationFromValues(
       4_000_000,
       "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaabbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb",
       200_00,
       "eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeffffffffffffffffffffffffffffffff"
     ),
+    ssz.electra.Attestation.defaultValue(),
   ];
 
   for (const [i, attestation] of testCases.entries()) {
     it(`attestation ${i}`, () => {
-      const bytes = ssz.phase0.Attestation.serialize(attestation);
+      const isElectra = "committeeBits" in attestation;
```

Comment:
add a new util to `types` package util:

```typescript
function isElectraAttestation(attestation: allForks.Attestation): attestation is electra.Attestation {
  return "committeeBits" in attestation;
}
```

or just make "TODO: use isElectraAttestation() util"
I'm creating/using that util in my PR

Reply:
> add a new util to `types` package util:
> 
> ```ts
> function isElectraAttestation(attestation: allForks.Attestation): attestation is electra.Attestation {
>   return "committeeBits" in attestation;
> }
> ```

Added 👍